### PR TITLE
964/versioning releasing

### PR DESCRIPTION
**Pull request by commjoen**
This PR covers #964 for word and pdf. It is ready to be merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <a href="https://leanpub.com/mobile-security-testing-guide"><img width=180px align="right" style="float: right;" src="Document/Images/mstg-cover-release-small.jpg"></a>
 
+
 # OWASP Mobile Security Testing Guide
+[![Creative Commons License](https://i.creativecommons.org/l/by-sa/3.0/88x31.png)](http://creativecommons.org/licenses/by-sa/3.0/)
```
Comment (TheDauntless) [inline README.md]: Add the URL titles

File: Document/0x02-Frontispiece.md [hunk 2]
```diff
@@ -77,9 +77,7 @@ Reviewers have consistently provided useful feedback through GitHub issues and p
 
 ### Others
 
-Many other contributors have committed small amounts of content, such as a single word or sentence (less than 50 additions). The full list of contributors is available on GitHub:
-
-https://github.com/OWASP/owasp-mstg/graphs/contributors
+Many other contributors have committed small amounts of content, such as a single word or sentence (less than 50 additions). The full list of contributors is available on [GitHub](https://github.com/OWASP/owasp-mstg/graphs/contributors Github).
```
Comment (TheDauntless) [inline Document/0x02-Frontispiece.md]: Add the URL title.

File: README.md [hunk 3]
```diff
@@ -58,16 +63,19 @@ This is the official GitHub Repository of the OWASP Mobile Security Testing Guid
 
 The release 1.0 of the MSTG was published in June 2018. You can get intermediate builds in multiple formats.
 
-1. Get the [e-book](https://leanpub.com/mobile-security-testing-guide-preview). The book is available for free, but you can choose to purchase it at a price of your choosing if you wish to support our project. All funds raised through sales of the e-book go directly into the project budget and will be used to fund production of future releases.
+1. Read it on [Gitbook](https://mobile-security.gitbook.io/mobile-security-testing-guide/). The book is automatically synchronized with the main repo. You can use the gitbook command line tool to generate PDF, epub, and other e-book formats, once you have checked out this repo.
```
Comment (TheDauntless) [inline README.md]: Add the URL title.

File: README.md [hunk 4]
```diff
@@ -58,16 +63,19 @@ This is the official GitHub Repository of the OWASP Mobile Security Testing Guid
...
-2. Read it on [Gitbook](https://mobile-security.gitbook.io/mobile-security-testing-guide/). The book is automatically synchronized with the main repo. You can use the gitbook command line tool to generate PDF, epub, and other e-book formats, once you have checked out this repo.
-
-To generate the epub version you can use the following command
+To generate the epub version you can use the following command:
 
 ```bash
 $ gitbook epub ./ Generated/MSTG.epub
 ```
 
+2. Check the [releases](URLHERE!). Here you can find a PDF, the sources zipped and a DocX document for any given tagged version. Please note that the documents are generated automatically per tag.
```
Comment (TheDauntless) [inline README.md]: Is this URL auto filled somehow, or did you forge to put the right one in?

File: README.md [hunk 5]
```diff
@@ -58,16 +63,19 @@ This is the official GitHub Repository of the OWASP Mobile Security Testing Guid
...
+
+3. Get the [e-book](https://leanpub.com/mobile-security-testing-guide-preview). The book is available for free, but you can choose to purchase it at a price of your choosing if you wish to support our project. All funds raised through sales of the e-book go directly into the project budget and will be used to fund production of future releases.
```
Comment (TheDauntless) [inline README.md]: Url title.

File: README.md [hunk 6]
```diff
@@ -76,6 +84,15 @@ $ cd owasp-mstg/Tools/
 $ ./generate_document.sh
 ```
 
+4. Clone the repository and run the [javascript pdf generator](https://github.com/OWASP/owasp-mstg/blob/master/Tools/genpdf.js) (requires [node 6](http://nodejs.org)). This produces a pdf file in the "Generated" subdirectory.
```
Comment (TheDauntless) [inline README.md]: URL Titles

File: README.md [hunk 7]
```diff
@@ -76,6 +84,15 @@ $ cd owasp-mstg/Tools/
...
+
+```bash
+$ git clone https://github.com/OWASP/owasp-mstg/
+$ cd owasp-mstg/Tools/
+$ npm install
+$ node genpdf.js
+```
+
 You can also use the [document index](https://rawgit.com/OWASP/owasp-mstg/master/Generated/OWASP-MSTG-Table-of-Contents.html) to navigate the master branch of the MSTG.
```
Comment (TheDauntless) [inline README.md]: URL Title

File: README.md (hunk 4 above)
Comment (TheDauntless) [inline README.md]: "sources zipped" -> "zipped sources", or "an archive containing the sources and a DocX...."

File: README.md [hunk 8]
```diff
@@ -91,6 +108,9 @@ Before you start contributing, please read our brief [style guide](https://githu
 If there's something you really want to see in the guide, or you want to suggest an improvement, create an issue [issue](https://github.com/OWASP/owasp-mstg/issues) or ping us on [Slack](https://owasp.slack.com/messages/project-mobile_omtg/details/).
 If the issue is accepted, we will schedule it into one of our [milestones](https://github.com/OWASP/owasp-mstg/milestones).
 
+### Translating
+We are currently in progress for making a new release every 6 months. Releases that have been tagged can then be translated into preferred languages.
```
Comment (TheDauntless) [inline README.md]: for -> of

File: Tools/package.json [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "publisher",
+  "version": "0.1.0",
+  "description": "Publisher used for creating pdfs of mstg and masvs",
+  "main": "genpdf.js",
+  "scripts": {
+    "test": "-h"
+  },
+  "author": "Jeroen Willemsen",
+  "license": "MIT",
+  "dependencies": {
```
Comment (TheDauntless) [inline Tools/package.json]: Can you fix the coffee-script dependency? 

> npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)

File: README.md (hunk 1 above)
Comment (commjoen) [reply README.md]: This is odd, i already have that. Changed "CC" in to "CC-by-SA 3"

File: Document/0x02-Frontispiece.md (hunk 2 above)
Comment (commjoen) [reply Document/0x02-Frontispiece.md]: done

File: README.md (hunk 3 above)
Comment (commjoen) [reply README.md]: Done!

File: README.md (hunk 4 above)
Comment (commjoen) [reply README.md]: Yes, when I wrote the readme, there was no release pushed yet. Fixed it now.

File: README.md (hunk 5 above)
Comment (commjoen) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (commjoen) [reply README.md]: done

File: README.md (hunk 7 above)
Comment (commjoen) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (commjoen) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (commjoen) [reply README.md]: done

File: Tools/package.json (hunk 9 above)
Comment (commjoen) [reply Tools/package.json]: coffeescript is used by the dependencies, I guess the warning will live until we can update the dependencies itself not referencing coffee-script ;-)

**Pull request merged**